src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/LineWrappingHandler.java

```diff
@@ -262,7 +262,8 @@ private void checkAnnotationIndentation(DetailAST atNode,
             if (isCurrentNodeCloseAnnotationAloneInLine
                     || node.getType() == TokenTypes.AT
                     && (parentNode.getParent().getType() == TokenTypes.MODIFIERS
-                        || parentNode.getParent().getType() == TokenTypes.ANNOTATIONS)) {
+                        || parentNode.getParent().getType() == TokenTypes.ANNOTATIONS)
+                    || parentNode.getType() == TokenTypes.MODIFIERS) {
```

Comment:
@pietern all the other checks are verifying that we are validating the annotation, `|| parentNode.getType() == TokenTypes.MODIFIERS` by itself is just saying we should work if it is the modifiers, regardless of annotations or not.
The whole method is annotation related, shouldn't this check be verifying a annotation somehow?

Reply:
I honestly don't remember, it's been about 2 years since I last used checkstyle :)

Reply:
sorry, I pinged the wrong person. Don't know how auto-complete picked you. Sorry.

@djydewang I was directing this at you.